descriptors/ModuleDescriptor-template.json

```diff
@@ -472,6 +472,14 @@
             "circulation.requests.item.put"
           ]
         },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inn-reach/transactions/{id}/itemhold/recall",
+          "permissionsRequired": ["inn-reach.d2ir.inn-reach-transaction.item.post"],
+          "modulePermissions": [
+            "circulation.requests.item.get"
```

Comment:
there should be permission for creating request as well

Reply:
Added